pytorch_toolkit/object_detection/tests/run_train_tests.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python3
+#
+# Copyright (C) 2020 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+
+""" This module contains unit tests. """
+
+import unittest
+
+
+def main():
+    testsuite = unittest.TestLoader().discover('tests', pattern='train_tests_*.py')
+    unittest.TextTestRunner(verbosity=1).run(testsuite)
```

Comment:
*_tests.py scripts always return 0

Add `sys.exit` to all *tests.py 
```
import sys
...
ret = not unittest.TextTestRunner(verbosity=1).run(testsuite).wasSuccessful()
sys.exit(ret)
```

Reply:
done